inbm/dispatcher-agent/dispatcher/sota/snapshot.py

```diff
@@ -58,6 +59,11 @@ def take_snapshot(self) -> None:
         """
         pass
 
+    @abstractmethod
+    def delete_snapshot(self) -> int:
+        """Deletes snapshots from the snapshot directory"""
```

Comment:
Abstract method comment should apply to all implementations. Here it is correct for Ubuntu but incorrect for Yocto. In my opinion this method belongs only in the Ubuntu/Debian class.